library/core/src/macros/mod.rs

```diff
@@ -1010,8 +1010,12 @@ pub(crate) mod builtin {
     #[rustc_builtin_macro]
     #[macro_export]
     macro_rules! format_args {
-        ($fmt:expr) => {{ /* compiler built-in */ }};
-        ($fmt:expr, $($args:tt)*) => {{ /* compiler built-in */ }};
+        ($fmt:expr) => {
+            $crate::hint::must_use({ /* compiler built-in */ })
```

Comment:
this doesn't do anything, same for all other builtin macros. you'd need to modify the compiler

Reply:
I am afraid this undergoing is way above my capabilities